### PR TITLE
Add more context

### PR DESCRIPTION
**Pull request by orestisfl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,30 @@ This repository contains:
 
 The current version of the web app for e-voting front-end is hosted at <https://vmn-webapp.azurewebsites.net/> (see instructions for updating this URL below).
 
+## Overview
+
+The general overview of the voting process is ass follows:
```
Comment (slarse) [inline README.md:18]: :)
```suggestion
The general overview of the voting process is as follows:
```

Comment (orestisfl) [reply README.md:18]: Haha, thanks

**Comment by monperrus** [comment]
@orestisfl there is a conflict here. could you have a look?

**Comment by orestisfl** [comment]
@monperrus Thanks, up

**Comment by monperrus** [comment]
Good, there are no more TODOs in the README. Thanks.

**Pull request merged**